blake2/src/blake2b.rs

```diff
@@ -1,8 +1,17 @@
-use digest::generic_array::typenum::U64;
+use digest::generic_array::typenum::{U16, U32, U64};
 use consts::BLAKE2B_IV;
 
-blake2_impl!(VarBlake2b, Blake2b, u64, u64x4, U64,
+blake2_impl!(VarBlake2b, FixedBlake2b, u64, u64x4, U64,
     32, 24, 16, 63, BLAKE2B_IV,
     "Blake2b instance with a variable output.",
     "Blake2b instance with a fixed output.",
 );
+
+/// Blake2b with 128 bit output
+pub type Blake2b128 = FixedBlake2b<U16>;
+/// Blake2b with 256 bit output
+pub type Blake2b256 = FixedBlake2b<U32>;
+/// Blake2b with 512 bit output
+pub type Blake2b512 = FixedBlake2b<U64>;
+/// Blake2b with 512 bit output
+pub type Blake2b = FixedBlake2b<U64>;
```

Comment:
Because you have used `struct $fix_state<U = $bytes> { .. }`, you don't have to introduce `FixedBlake2b`. You can write `type Blake2b256 = Blake2b<U32>;` and remove `Blake2b512` as a redundant one. Same goes for `Blake2s`

Reply:
Hm, I will need to look into this... This behavior may create issues in `blowfish` crate which switched to default type parameters recently.